niworkflows/tests/conftest.py

```diff
@@ -27,11 +26,6 @@ def _run_interface_mock(objekt, runtime):
     return runtime
 
 
-def pytest_runtest_setup(item):
```

Comment:
Are we not chdir'ing before every test now?

Reply:
Do we need to? I can restore it, since the problem was actually pytest-cov, but I don't see the purpose.

Reply:
Sure, it was there just for prophylaxis of testing, but there's no particular reason to have it. If removing it makes coverage collecting easier there's no point in restoring it :+1:

Reply:
It doesn't actually seem to have any effect. I'll restore it to declutter this PR and add a note to reconsider later.